app-libs/stf/src/cli.rs

```diff
@@ -35,6 +35,26 @@ const KEYSTORE_PATH: &str = "my_trusted_keystore";
 pub fn cmd<'a>(
 	perform_operation: &'a dyn Fn(&ArgMatches<'_>, &TrustedOperation) -> Option<Vec<u8>>,
 ) -> MultiCommand<'a, str, str> {
+	macro_rules! get_layer_two_nonce {
```

Comment:
Is a macro necessary because of the `perform_operation` fn? Or why are you using a macro here?

Reply:
exactly